DopplerCustomDomain/appsettings.int.json

```
@@ -0,0 +1,17 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "PublicKeysFolder": "public-keys",
+  "TraefikConfiguration": {
+    "ServicesMapping": {
+      "relay-tracking": "",
```

I would put a name, then it can be changed (the same for other environments):

```suggestion
      "relay-tracking": "relay_tracking_api_int@docker",
```